backend/data/processors/conditions_preprocessing.py

```diff
@@ -432,6 +442,134 @@ def map_word_to_program_type(processed, regex_word, type):
         flags=re.IGNORECASE,
     )  # hard to capture a generic case?
 
+#adds handbook notes and removes phrasing
+def remove_extraneous_comm_data(conditions, processed):
```

Comment:
Is the `conditions` argument being used anywhere in this function?

Reply:
nope forgot to delete this thanks